dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -1199,6 +1199,19 @@
             "name": "Association",
             "modelAsString": false
           }
+        },
+        "temporality": {
```

Comment:
Where does this get used? It's not referenced.

Reply:
As with the other assertion types (_conditionality_, _certainty_, _association_), _temporality_ is potentially used for adding contextual meaning to a healthcare entity. It may be used in the (optional) "assertion" property of the _HealthcareEntity_ model.
It is indeed not meant to be explicitly referenced elsewhere in the API spec.